ui/src/app/ErrorInfo/ErrorInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Popover, Icon, Tooltip } from '@patternfly/react-core';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+import { DeploymentStepStatusData } from '@app/apis/types';
+
+interface IErrorInfoProps {
+  stepStatusData: DeploymentStepStatusData;
+}
+
+export const ErrorInfoPopover = ({ stepStatusData }: IErrorInfoProps) => {
+  const borders = {
```

Comment:
Let's move this out of the component and into a css file. Then let's add class names.